native/python312/Makefile

```diff
@@ -1,5 +1,5 @@
 PKG_NAME = python312
-PKG_VERS = 3.12.9
+PKG_VERS = 3.12.10
```

Comment:
Also worth updating `setuptools==78.1.0` although let pip stay at current version as v25.x as a lot more changes that we're not ready yet to handle entirely. That would apply for pyth312+313 (and the other PR for 311)